docs/schemas.md

```diff
@@ -162,7 +162,13 @@ If students are entering something into a text block, be sure to follow these co
 The output if the `data-wait-for-event="eventnameY"` attribute is removed and the user submits an answer.
```

Comment:
Based on this text, I would expect the image that follows to depict the status of the UI when a user submits an answer. You've opted for showing active, submitting without providing a response, and "correct" submission. Do you think we should add additional text to clarify the two additional states of the `UserInputBlock`?

Reply:
I'll go back to having the submitted state only,  to be more consistent with what we had and make the documentation less verbose.